src/common/sync-box/sync-box.test.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { observe, runInAction } from "mobx";
+import type { ApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
+import { getApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
+import createSyncBoxInjectable from "./create-sync-box.injectable";
+import { flushPromises } from "../test-utils/flush-promises";
+import type { SyncBox } from "./sync-box-injection-token";
+
+describe("sync-box", () => {
+  let applicationBuilder: ApplicationBuilder;
+
+  beforeEach(() => {
+    applicationBuilder = getApplicationBuilder();
+
+    applicationBuilder.dis.mainDi.register(someInjectable);
+    applicationBuilder.dis.rendererDi.register(someInjectable);
+  });
+
+  // TODO: Separate starting for main application and starting of window in application builder
+  xdescribe("given application is started, when value is set in main", () => {
```

Comment:
Note: This is unacceptable but had to do it for now to stop blocking other development regarding to auto update.

Reply:
Some CS-geeks might object, but those may be the same thing :)

Fixed and resolved.

Reply:
Resolved for agreement.